pkg/gitops/gitops.go

```diff
@@ -46,6 +48,10 @@ func Setup(k8sRestConfig *rest.Config, k8sClientSet kubeclient.Interface, cfg *a
 		return err
 	}
 
+	if fluxSSHKey != nil {
+		return deploykey.ForCluster(cfg).Put(context.Background(), *fluxSSHKey)
+	}
+
 	logger.Info(userInstructions)
```

Comment:
If the user doesn't set the token env var it should be prompted to set the key manually. But it will never get to it.

Reply:
Addressed in 754fe0b3. Does it look good to you now?

Reply:
Have you tested this without providing a token? It should prompt the user to manually set the deploy key.

Reply:
Yes. And it works as intended for me.

Here's the log from `eksctl create -f cluster.yaml` when GITHUB_TOKEN unset:

```
*snip*
[ℹ]  created "flux:Deployment.apps/memcached"
[ℹ]  created "CustomResourceDefinition.apiextensions.k8s.io/helmreleases.helm.fluxcd.io"
[ℹ]  created "flux:Deployment.apps/helm-operator"
[ℹ]  Waiting for Helm Operator to start
[ℹ]  Helm Operator started successfully
[ℹ]  see https://docs.fluxcd.io/projects/helm-operator for details on how to use the Helm Operator
[ℹ]  Waiting for Flux to start
[ℹ]  fetching public SSH key from Flux
[ℹ]  Flux started successfully
[ℹ]  see https://docs.fluxcd.io/projects/flux for details on how to use Flux
[ℹ]  Committing and pushing manifests to git@github.com:mumoshu/gitops-demo.git
[ℹ]  Nothing to commit (the repository contained identical files), moving on
Everything up-to-date
[ℹ]  Flux will only operate properly once it has write-access to the Git repository
[ℹ]  GITHUB_TOKEN is not set. Please set it so that eksctl is able to create and delete GitHub deploy key from Flux SSH public key
[ℹ]  please configure git@github.com:mumoshu/gitops-demo.git so that the following Flux SSH public key has write access to it
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQC9u7F63s57eFD7mHmOKPMR2Czi76G4vAYExnlH0jT/SUnkPWaj2hH+OPzJgVPIKk2VaskEyLf4D8z5CRy1FVjtan+gf2L9yAY9KU2d+LXo7qdSyT/XauUfdnkWN5En30zDCSQtbY/lvoh3SXuj2GL6FB/QLqpPH3ULWpswOXeoIiLoig1SFPgtuFwy66whyOx8o0MDHHr9MVrQtiPQWtCP8/x51puuFqXQ25R7srXccEYhYo1bk09g03gk7o4shGaZx5eOSLliFgIH6C7nk7j4OGnlLtrOb/atCD79NzboOvUA6EjEzs/3j8D2DJbY08ahZJ/ryOIStT7BB6a6o+wOy3aHLs3wk2pOsHiQL6NyDLr/3X4wrR0DAUyeLBLqRBqa/oNL50EfyJ7h24BHi22BK8PMmAKxGQSDKaAtq6wQ4xKptohy37utROrvTmIcrc4NgwRCwvyy8svegv68HrPZlK4hUkjCPw6IPhvIlPdhDDER8qx8ziTgB0ayEXh1kiM= root@flux-56b96cc9fb-d5r28
```

Reply:
I believe the last line `please configure git@github.com:mumoshu/gitops-demo.git so that the following Flux SSH public key has write access to it` is the prompt you asked.

It has been there before this change, and will remain there after this change.

Reply:
Thanks for testing it